horizons/main.py

```diff
@@ -445,36 +445,45 @@ def _load_cmd_map(savegame, ai_players, force_player_id=None):
 	return True
 
 def _find_matching_map(name_or_path, savegames):
-	"""*name_or_path* is either a map/savegame name or path to a map/savegame file."""
+	"""Find a map by name or path specifiec by user.
+	@param name_or_path: either a map/savegame name or path to a map/savegame file
```

Comment:
If you introduce these kind of variable declarations here, might as well make them complete and add one for `savegames`.